provision/provision-site.sh

```diff
@@ -76,6 +76,11 @@ function vvv_provision_site_nginx() {
   sed -i "s#{vvv_path_to_site}#${VM_DIR}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
   sed -i "s#{vvv_site_name}#${SITE}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
   sed -i "s#{vvv_hosts}#${VVV_HOSTS}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
+
+  if [ 'php' != "${NGINX_UPSTREAM}" ] && [ ! -f "/etc/nginx/upstreams/${NGINX_UPSTREAM}.conf" ]; then
```

Comment:
We don't need the first bit as `php` isn't a valid upstream value, just the final file check for the `.conf` file is enough

Reply:
I initially had it like this, but covered for php, since that was the default in Vagrantfile. (read other thread for context and links to references)

Reply:
hmmm perhaps we need to double check `php` works as a value